client/components/deposits-status/index.tsx

```diff
@@ -62,31 +64,51 @@ const DepositsStatusDisabled: React.FC< DepositsStatusProps > = ( props ) => {
 const DepositsStatusSuspended: React.FC< DepositsStatusProps > = ( props ) => {
 	const { iconSize } = props;
 
-	const learnMoreHref =
-		'https://woocommerce.com/document/woopayments/payouts/why-payouts-suspended/';
-
-	const description = createInterpolateElement(
+	const description =
 		/* translators: <a> - suspended accounts FAQ URL */
-		__(
-			'Temporarily suspended (<a>learn more</a>)',
-			'woocommerce-payments'
-		),
-		{
-			a: (
-				// eslint-disable-next-line jsx-a11y/anchor-has-content
-				<a
-					href={ learnMoreHref }
-					target="_blank"
-					rel="noopener noreferrer"
-				/>
-			),
-		}
-	);
+		__( 'Temporarily suspended', 'woocommerce-payments' );
 
 	return (
 		<span className={ 'account-status__info__yellow' }>
 			<GridiconNotice size={ iconSize } />
 			{ description }
+			<HoverTooltip
+				className="wcpay-tooltip--click__tooltip"
+				maxWidth={ '300px' }
+				content={ interpolateComponents( {
+					mixedString: sprintf(
+						/* translators: 1: WooPayments */
+						__(
+							// eslint-disable-next-line max-len
+							'After the information review your account was temporarily suspended. {{learnMoreLink}}Learn more{{/learnMoreLink}}',
```

Comment:
I think we need a coma after "review":

```
After the information review, your account was temporarily suspended.
```

Reply:
Thanks, fixed! https://github.com/Automattic/woocommerce-payments/pull/10214/commits/5dacdad220c5255687c662c420a3de961a10faf0